packages/server/graphql/mutations/removeReflectTemplate.ts

```diff
@@ -40,20 +39,15 @@ const removeReflectTemplate = {
 
     // VALIDATION
     const {teamId} = template
-    // Will convert to PG by Mar 1, 2023
-    const {templates, settings} = await r({
-      templates: r
-        .table('MeetingTemplate')
-        .getAll(teamId, {index: 'teamId'})
-        .filter({isActive: true, type: 'retrospective'})
-        .orderBy('name')
-        .coerceTo('array') as unknown as ReflectTemplate[],
-      settings: r
+    const [templates, settings] = await Promise.all([
+      dataLoader.get('meetingTemplatesByType').load({meetingType: 'retrospective', teamId}),
```

Comment:
We can also clear the cache in other places like this (there are few in this PR)